test/Microsoft.Framework.Runtime.FunctionalTests/project.json

```diff
@@ -0,0 +1,14 @@
+{
+	"dependencies": {
+		"Microsoft.AspNet.Testing": "1.0.0-*",
```

Comment:
I guess I can remove that package and let the tests run on mono even though the feature is not available on that platform. We should consider doing something that allows us to use test extensions in kruntime